app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -388,6 +391,27 @@ class SuggestedEditsTasksFragment : Fragment() {
         }
     }
 
+    private fun showSurveyDialog() {
+        if (!Prefs.contributionsDashboardSurveyDialogShown) {
+            Handler(Looper.getMainLooper()).postDelayed({
```

Comment:
Can we use `binding.tasksContainer.postDelayed` for this?

Reply:
I don't think this is possible if we move the function to ContributionsDashboardHelper.